apps/elasticsearch/templates/services/app/configs/set-index-template.sh.j2

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+MAX_RETRY=20
+RETRY_DELAY=5
+
+while ! curl --output /dev/null --silent --max-time 1 --connect-timeout 1 --head --fail "http://elasticsearch.{{project_name}}.svc:{{ elasticsearch_api_port }}/"
+do
+    echo "Waiting for ES service to be up..."
+    ((c++)) && ((c==$MAX_RETRY)) && break
+    sleep $RETRY_DELAY
+done
+
+set -e
+
+curl -fssL -X PUT "http://elasticsearch.{{project_name}}.svc:{{ elasticsearch_api_port }}/_template/default" -H 'Content-Type: application/json' -d'
+{
+{% if elasticsearch_image_tag is version('6.0', '<') %}
```

Comment:
Didn't know about the `version` comparison :clap:

Reply:
Neither do I, but I was happy to find it :smile: